lib/tests/streamlit/elements/arrow_table_test.py

```diff
@@ -128,16 +127,3 @@ def test_table_uses_convert_anything_to_df(self):
 
             st.table(df)
             convert_anything_to_df.assert_called_once()
-
-    @patch(
```

Comment:
nit: its using the exactly same logic in the background. We don't need to test this two times for dataframe & table.